composer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "drupal/media_entity_image",
+  "type": "drupal-module",
+  "description": "Local images integration for media entity.",
+  "keywords": ["Drupal"],
+  "license": "GPL-2.0+",
+  "homepage": "https://github.com/drupal-media/media_entity_image",
+  "minimum-stability": "dev",
+  "support": {
+    "issues": "https://github.com/drupal-media/media_entity_image/issues",
+    "source": "https://github.com/drupal-media/media_entity_image"
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/drupal-media/media_entity"
+    }
+  ],
+  "require": {
+    "drupal/media_entity": "dev-8.x-1.x"
```

Comment:
Checked how commerce does this and it seems that we should require core too. Would be also nice to update composer.json in media_entity.